src/main.rs

```diff
@@ -249,35 +258,70 @@ fn exec(cmd: Command) -> Result<String> {
             unstable_options,
         } => {
             let manifest_path = ManifestPath::try_from(manifest_path.as_ref())?;
-            let dest_wasm = cmd::build::execute(
+            if *(skip_metadata) {
+                let dest_wasm = cmd::build::execute(
+                    &manifest_path,
+                    verbosity.try_into()?,
+                    unstable_options.try_into()?,
+                    true,
+                )?
+                .expect("dest_wasm must exist");
+                return Ok(format!(
+                    "\nYour contract's code is ready. You can find it here:\n{}",
+                    dest_wasm.display().to_string().bold()
+                ));
+            }
+
+            let metadata_result = cmd::metadata::execute(
                 &manifest_path,
                 verbosity.try_into()?,
+                *skip_bundle,
                 unstable_options.try_into()?,
             )?;
+            let maybe_bundle = if !*(skip_bundle) {
+                format!(
+                    "\nYour contract bundle (code + metadata) is ready. You can find it here:\n{}",
+                    metadata_result
+                        .bundle_file
+                        .expect("bundle file must exist")
+                        .display()
+                        .to_string()
+                        .bold()
+                )
+            } else {
+                "".to_string()
+            };
             Ok(format!(
-                "\nYour contract is ready. You can find it here:\n{}",
-                dest_wasm.display().to_string().bold()
+                "\nYour contract's code is ready. You can find it here:\n{}
+                \nYour contract's metadata is ready. You can find it here:\n{}{}",
+                metadata_result.wasm_file.display().to_string().bold(),
+                metadata_result.metadata_file.display().to_string().bold(),
+                maybe_bundle,
             ))
         }
-        Command::GenerateMetadata {
+        Command::Check {
             manifest_path,
             verbosity,
             unstable_options,
         } => {
             let manifest_path = ManifestPath::try_from(manifest_path.as_ref())?;
-            let metadata_file = cmd::metadata::execute(
-                manifest_path,
+            let maybe_dest_wasm = cmd::build::execute(
+                &manifest_path,
                 verbosity.try_into()?,
                 unstable_options.try_into()?,
+                false,
             )?;
-            Ok(format!(
-                "Your metadata file is ready.\nYou can find it here:\n{}",
-                metadata_file.display()
-            ))
+            assert!(maybe_dest_wasm.is_none(), "no dest_wasm should exist");
+            Ok(format!("\nYour contract's code was built successfully."))
         }
+        Command::GenerateMetadata {} => Err(anyhow::anyhow!(format!(
+            "Command deprecated, use `cargo contract build` instead"
+        ))),
```

Comment:
```suggestion
        Command::GenerateMetadata {} => Err(anyhow::anyhow!(
            "Command deprecated, use `cargo contract build` instead"
        )),
```

The anyhow macro supports formatting syntax, already.

Reply:
Nice, thx!